ct.proto

```diff
@@ -39,6 +40,11 @@ message File {
   string ext = 2;
 }
 
+message FileWithOffset {
+	string filePath = 1;
```

Comment:
Rename this to - "FileForConversion", the first field of that will not be a "string filepath" but rather a "File" itself.